### PR TITLE
add the possitility to choose the symfony version by --symfony-versio…

### PR DESCRIPTION
**Pull request by whiskeylotus**
- If applied, this commit will allow to choose the version of symfony by using --symfony-version=3.4.4 (does not support new architecture of symfony 4 neither symfony 4), update some packages versions (to be updated) and add test about the new feature.

- This change is made to be updated to the last version of symfony by default but also to be able to choose the version needed by your project.

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -50,6 +53,15 @@ while [ "$1" != "" ]; do
 	shift
 done
 
+if [ -z "$SYMFONY_VERSION" ]; then
+	SYMFONY_VERSION="^3.4"
+fi
+
+if [ $(echo "$SYMFONY_VERSION >= 4" | bc) -eq "1" ]; then
```
Comment (mageekguy) [inline install.sh]: `if [[ "$SYMFONY_VERSION" =~ ^[0-9]+(.[0-9]+)*$ && ! "$SYMFONY_VERSION" =~ ^([456789][0-9]*|[1-9][0-9]+|[0-2]).? ]]; then`

File: install.sh [hunk 2]
```diff
@@ -50,6 +53,15 @@ while [ "$1" != "" ]; do
...
+	echo Symfony 4 not yet supported, please install as default for the last LTS supported aka 3.4
```
Comment (mageekguy) [inline install.sh]: `Symfony version less than 3 or greater than 3 is not currently supported` ?

**Comment by whiskeylotus** [comment]
I've corrected your review !

**Comment by mageekguy** [comment]
Thanks for your job, @whiskeylotus!

**Pull request merged**